File: 1. Arrays & Hashing/1. Contains Duplicate/long.py
# Runtime 420ms
# Beats 86.76%

class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        listLength = len(nums)
        setLength = len(set(nums))
        return True if listLength > setLength else False

File: 1. Arrays & Hashing/1. Contains Duplicate/long.java
// Runtime 19ms
// Beats 37.35%

// Memory 59.93MB
// Beats 19.64%

class Solution {
    public boolean containsDuplicate(int[] nums) {
        Set<Integer> numsSet = new HashSet<>();
        for(int num : nums) {
            numsSet.add(num);
        }
        return numsSet.size() != nums.length;
    }
}